hoodie-client/src/main/java/com/uber/hoodie/io/HoodieMergeHandle.java

```diff
@@ -156,9 +164,21 @@ private String init(String fileId, Iterator<HoodieRecord<T>> newRecordsItr) {
       // update the new location of the record, so we know where to find it next
       record.setNewLocation(new HoodieRecordLocation(commitTime, fileId));
     }
+    logExtra();
     return partitionPath;
   }
 
+  private void logExtra() {
```

Comment:
same comment. can we do a single log line? and get rid of this method?

Reply:
yup